elide-model-config/src/main/java/com/yahoo/elide/modelconfig/parser/handlebars/HandlebarsHydrator.java

```diff
@@ -36,6 +37,23 @@
         {"\n", " " }
     });
 
+    private final Handlebars handlebars;
+    private final HandlebarsHelper helper;
+
+    public HandlebarsHydrator(StaticModelsDetails staticModelDetails) {
+        TemplateLoader loader = new ClassPathTemplateLoader("/templates");
+        this.helper = new HandlebarsHelper(staticModelDetails);
+        this.handlebars = new Handlebars(loader).with(MY_ESCAPING_STRATEGY);
+        this.handlebars.registerHelpers(ConditionalHelpers.class);
+        this.handlebars.registerHelper(AssignHelper.NAME, AssignHelper.INSTANCE);
+        this.handlebars.registerHelpers(helper);
+    }
+
+    public HandlebarsHydrator() {
+        this.handlebars = new Handlebars();
+        this.helper = null;
```

Comment:
What is the use case for this constructor?

Reply:
hydrateTableTemplate and hydrateSecurityTemplate methods use helper methods and I had to pass staticModelDetails to helper class. 
Default constructor is for when to use just hydrateConfigTemplate method that doesn't use helper class.

Reply:
Seems like it is safer to only have one fully initialized constructor.  Why not just use the same one throughout (even if config template doesn't need the static info)?